danger-spelling.gemspec

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/HamstringAssassin/danger-spelling/blob/8b5ef5e47ae075e09604a70ce13103f01902f481/danger-spelling.gemspec#L1"></span><code>required_ruby_version</code> should be specified.</td>
    </tr>
  </tbody>
</table>